src/ota.rs

```diff
@@ -218,6 +273,7 @@ pub struct EspOtaUpdateFinished<'a> {
 }
 
 impl EspOtaUpdateFinished<'_> {
+    /// Sets the boot partition to the newly updated app partition.
```

Comment:
Note that the user must still reboot for the updated app partition to go into effect

Reply:
I added "The app will be run on the next boot.". Does that sound OK?